packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -168,6 +168,7 @@ const DropdownMenuTrigger = React.forwardRef<DropdownMenuTriggerElement, Dropdow
           disabled={disabled}
           {...triggerProps}
           ref={composeRefs(forwardedRef, context.triggerRef)}
+          onClick={composeEventHandlers(props.onClick, () => context.onOpenChange(true))}
```

Comment:
So the idea is to let the `onClick` handle the spacebar because it's native, even though the pointer side of things won't ever execute given we do it in pointer down?

I don't mind that much, but I wonder if it's perhaps more confusing this way (less explicit) as you come back later down and see, oh we're already doing the `onPointerDown`, I can remove the `onClick`.

Thoughts?

Reply:
`onClick` handles "anything that is considered a click" so for buttons that includes space *keyup* and enter *keydown* which we had wrong before (we did keydown for both). I think it's better to just let the browser handle it to ensure we're not losing any of these little details.

I agree that it is a bit confusing now though that we also have the `onPointerDown` logic, so maybe I should add a comment?

Reply:
Yeah I think adding a comment in a few places would be great.

Reply:
Done 🙂  Let me know if you want me to reword anything 🙏